src/main_test.rs

```diff
@@ -0,0 +1,19 @@
+#[cfg(test)]
+
+use std::process::Command;
+
+#[test]
+fn test_run_mowl_init_ok() {
```

Comment:
These tests does not seem to work on windows. I would recommend to test the binary not with system commands, but we should test the separate functions directly.